packages/forge2d/example/web/box_test.dart

```diff
@@ -69,5 +61,8 @@ class BoxTest extends Demo {
 }
 
 void main() {
-  BoxTest.main();
+  final boxTest = BoxTest();
+  boxTest.initialize();
+  boxTest.initializeAnimation();
+  boxTest.runAnimation();
```

Comment:
since we always call the same 3 methods, shouldn't `Demo` expose a helper for that?

Reply:
Yeah, there are looots of fixes to do here, but this was just a minor side fix in the PR (to remove `static main`)